nengo_loihi/splitter.py

```diff
@@ -289,13 +289,10 @@ def split_host_to_chip(networks, conn):
     networks.add(receive2post, "chip")
 
     logger.debug("Creating NIF ensemble for %s", conn)
-    ens = nengo.Ensemble(
-        2 * dim, dim,
-        neuron_type=NIF(tau_ref=0.0),
-        encoders=np.vstack([np.eye(dim), -np.eye(dim)]),
-        max_rates=np.ones(dim * 2) * networks.max_rate,
-        intercepts=np.ones(dim * 2) * -1,
-        add_to_container=False)
+    if networks.inter_neurons is None:
+        raise BuildError("Interneurons must be specified for host->chip "
+                         "connection.")
+    ens = networks.inter_neurons.get_ensemble(dim)
```

Comment:
If `get_ensemble` is expected to exist for all `Interneuron` instances, it should be part of the top-level `Interneuron` base class (currently it does not).